mvp_project/README.md

```diff
@@ -1,109 +1,183 @@
+
+2. **Добавление мультимедиа**  
+   _Как пользователь, я хочу загружать фото и видео для визуализации записей._  
+   - **Контекст**: Загрузка файлов, их просмотр перед сохранением.  
+   - **Технически**: FastAPI отправляет файлы в облачное хранилище (AWS S3).  
+
+3. **Организация записей**  
+   _Как пользователь, я хочу упорядочивать записи по дням и местам._  
+   - **Контекст**: Добавление даты и местоположения к записи.  
+   - **Технически**: Записи связываются с дневником через внешние ключи.  
 
-### 9. Автоматическое описание мест
-- **Как пользователь**, я хочу, чтобы приложение распознавало достопримечательности на моих фотографиях и добавляло их описание, чтобы экономить время.
-  - **Контекст:** При загрузке фотографии приложение предлагает автоматическое описание.
-  - **Технически:** Используются API для анализа изображений, например Google Vision API.
+4. **Автоматическое отслеживание маршрутов**  
+   _Как пользователь, я хочу автоматически записывать маршруты с использованием GPS._  
+   - **Контекст**: Включение GPS для реального времени.  
+   - **Технически**: FastAPI принимает данные через WebSocket или POST-запросы.  
 
-### 10. Рекомендации по маршрутам
-- **Как пользователь**, я хочу получать рекомендации о местах, которые можно посетить, на основе моих интересов, чтобы лучше планировать поездки.
-  - **Контекст:** После загрузки маршрута пользователь видит предложенные направления.
-  - **Технически:** Рекомендательная система использует алгоритмы машинного обучения, анализируя данные других пользователей.
+---
+
+### **Категория 2: Социальные взаимодействия**
+1. **Обмен дневниками**  
+   _Как пользователь, я хочу делиться дневниками через уникальную ссылку._  
+   - **Контекст**: Генерация ссылки с токеном доступа.  
+   - **Технически**: FastAPI предоставляет временный токен доступа.  
+
+2. **Лента активности**  
+   _Как пользователь, я хочу видеть обновления друзей._  
+   - **Контекст**: Лента показывает новые записи и дневники друзей.  
+   - **Технически**: Пагинированный список через FastAPI.  
 
 ---
 
-## Категория 5: Управление приватностью
+### **Категория 3: Интерактивные функции**
+1. **Визуализация маршрутов**  
+   _Как пользователь, я хочу видеть свои маршруты на карте._  
+   - **Контекст**: Карта отображает точки с медиафайлами.  
+   - **Технически**: Использование Google Maps API или Mapbox.  
 
-### 11. Настройки видимости
-- **Как пользователь**, я хочу выбирать, кто может видеть мои дневники, чтобы контролировать доступ к ним.
-  - **Контекст:** В настройках дневника пользователь устанавливает уровень видимости: публичный, только по ссылке, личный.
-  - **Технически:** FastAPI проверяет права доступа при каждом запросе к дневнику.
+2. **Создание видеоисторий**  
+   _Как пользователь, я хочу создавать видеоистории из записей._  
+   - **Контекст**: Выбор шаблона, добавление музыки, экспорт видео.  
+   - **Технически**: Генерация видео через FFmpeg.  

```

Comment:
Managing files and editing it in a web view, can be challenging. 

For now I would say uploading and displaying. To have templates and editing will increase the size of the MVP